config/settings/base.py

```diff
@@ -33,6 +33,14 @@
 DATABASES["default"]["ATOMIC_REQUESTS"] = True
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+SECONDARY_DB_ALIAS = "default"
```

Comment:
I don't think we should use a default value that represents a broken setup. It should either be `None` or a valid alias.

Reply:
Should this be in the env?

Reply:
None makes sense. It doesn't need to be in the env, since it can be inferred based on `SECONDARY_DATABASE_URL`